--- origsrc/ocaml-speex-0.1.0/src/META.in	2009-02-17 12:37:40.000000000 -0600
+++ src/ocaml-speex-0.1.0/src/META.in	2009-03-03 14:52:20.226625000 -0600
@@ -1,6 +1,6 @@
 name="speex"
 version="@VERSION@"
 description="OCaml bindings for speex"
-requires="@requires@"
+requires="ogg"
 archive(byte)="speex.cma"
 archive(native)="speex.cmxa"
--- origsrc/ocaml-speex-0.1.0/src/Makefile.in	2009-02-17 12:37:40.000000000 -0600
+++ src/ocaml-speex-0.1.0/src/Makefile.in	2009-03-03 14:53:42.351625000 -0600
@@ -27,13 +27,14 @@ SOURCES = speex_stubs.c speex.ml speex.m
 RESULT = speex
 OCAMLDOCFLAGS = -stars
 LIBINSTALL_FILES = $(wildcard *.mli *.cmi *.cma *.cmxa *.cmx *.a *.so)
-ACLIBS = @LIBS@ @speex_LIBS@ @ogg_LIBS@
+ACLIBS = @LIBS@ ogg_stubs @speex_LIBS@ @ogg_LIBS@
 LDFLAGS = @LDFLAGS@ @speex_LDFLAGS@ @ogg_LDFLAGS@
 CLIBS = $(ACLIBS:-l%=%)
 CC = @CC@
 CFLAGS = @CFLAGS@ @speex_CFLAGS@ @ogg_CFLAGS@  -Wall -DCAML_NAME_SPACE
 CPPFLAGS = @CPPFLAGS@
 INCDIRS = @INC@ @OCAMLOGG_INC@
+LIBDIRS = @OCAMLOGG_INC@
 
 all: $(BEST)
 
